exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetRecordWriter.java

```diff
@@ -292,7 +293,12 @@ private Type getType(MaterializedField field) {
         }
         return new GroupType(dataMode == DataMode.REPEATED ? Repetition.REPEATED : Repetition.OPTIONAL, field.getName(), types);
       case LIST:
-        throw new UnsupportedOperationException("Unsupported type " + minorType);
+        MaterializedField elementField = field.getChildren().iterator().next();
```

Comment:
I assume you are expecting here `$data$` field, but in some cases, list filed may also have `$offsets$` child. Should we obtain a child with the required name here (`BaseRepeatedValueVector.DATA_VECTOR_NAME`)?

Reply:
yes, good idea, thanks:)